configs/common/PrefetcherConfig.py

```
@@ -5,118 +5,175 @@
 from m5.objects.Prefetcher import XSPhysicalSmallBOP, XSVirtualLargeBOP
 
 
+DEFAULT_ENABLE_PF_BUFFER = True
+
+
 def _get_hwp(hwp_option):
     if hwp_option == None:
```

Ruff (0.15.15)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)